demo/backstop-common.js

```diff
@@ -112,7 +112,7 @@ module.exports = function backstopCommon(baseUrl) {
         viewports: [{ label: 'desktop', width: 800, height: 200 }],
       },
       {
-        label: 'Components/Header',
+        label: 'Components/Header (search open)',
```

Comment:
This previously had the same name as another scenario, surprised it didn't clash before!